lib/commit-validator.js

```diff
@@ -8,13 +9,21 @@ class CommitValidator {
     ];
 
     this.rules = {
+      'body-leading-blank': [1, 'always'],
+      'footer-leading-blank': [1, 'always'],
+      'scope-enum': [2, 'always', options.scopes],
+      'scope-empty': [2, 'never'],
+      'subject-case': [2, 'never', ['start-case', 'pascal-case']],
+      'subject-empty': [2, 'never'],
+      'subject-full-stop': [2, 'never', '.'],
+      'subject-start-jira': [2, 'always'],
+      'type-case': [2, 'always', 'lower-case'],
```

Comment:
@icatalina I added some additional rules from [`config-conventional`](https://github.com/conventional-changelog/commitlint/blob/20946edabc7ef05d6bd576d4167cb9f711d1c035/%40commitlint/config-conventional/index.js#L3). The primary ones to be aware about are the following:

- `body-leading-blank`: ensures a leading blank line before writing your commit body
- `footer-leading-blank`: same as above, but with the footer of the commit
- `subject-case`: needed to ensure you don't write something like `Big-123 ...` or `TickEt-456` at the start of your subject.
- `subject-full-stop`: ensures you don't end the subject with a period

Let me know if those don't make sense to add into this PR.

Reply:
they sound great